datamodels/2.x/itop-portal-base/portal/src/Form/ObjectFormManager.php

```diff
@@ -85,6 +83,15 @@ class ObjectFormManager extends FormManager
 	 */
 	protected $aHiddenFieldsId = array();
 
+	/**
+	 * Replace container. Allow access to others applications services.
+	 *
+	 * @var ObjectFormHandlerHelper $oFormHandlerHelper
+	 * @since 3.1
+	 */
```

Comment:
```suggestion
	/**
	 * @var ObjectFormHandlerHelper $oFormHandlerHelper
	 * @since 3.1.0 Replace container. Allow access to others applications services.
	 */
```

Reply:
Also add bug reference ? 

```suggestion
	/**
	 * @var ObjectFormHandlerHelper $oFormHandlerHelper
	 * @since 3.1.0 N°5281 Replace container. Allow access to others applications services.
	 */
```